_plugins/team.rb

```diff
@@ -2,12 +2,13 @@ module Jekyll
   module TeamFilter
     def team_photo(name)
       person = finder(name)
+      baseurl = Jekyll.sites[0].config['baseurl']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.